homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { todoList } from './services/todoContents';
+import Todo from './components/todo';
+import TodoStatic from './components/todoStatic';
+
+function App() {
+  const [todoListState, setTodoListState] = useState(todoList);
+
+  const handleDone = id => {
+    const todoIndex = todoListState.findIndex(todo => todo.id === id);
+    const todo = { ...todoListState[todoIndex] };
+    todo.done = !todo.done;
+    const todoList = [...todoListState];
+    todoList[todoIndex] = todo;
+    setTodoListState([...todoList]);
```

Comment:
`todoList` is already a copy of your initial state, not need to make another copy before setting it as your new state

Reply:
OK, I'll change it, I think I don't even need to assign it to a variable, can I do it like this?
```
  todoListState[todoIndex] = todo;
  setTodoListState([...todoListState]);
```